epistles-v2/original/05-epistles.adoc

```diff
@@ -553,6 +554,7 @@ Amen.
 [.signed-section-signature]
 G+++.+++ F.
 
+SSCC
```

Comment:
whoops, looks like you only half got this line done 😄

Reply:
Also, above here, line 533 (`Swarthmore, in Lancashire...`) should be a SSCC as well.

Reply:
Also, line 665 (`Swarthmore, the 17th of the 3d month...`) also needs to be a SSCC. 👍

Reply:
I wonder if I thought i had finished this chapter and actually didn't.....sorry!!!